src/main/blackbox/blackbox_io.c

```diff
@@ -470,7 +470,7 @@ bool blackboxDeviceOpen(void)
      *
      * 9 / 1250 = 7200 / 1000000
      */
-    blackboxWriteChunkSize = MAX((masterConfig.looptime * 9) / 1250, 4);
+    blackboxWriteChunkSize = MAX((GUARD_LOOPTIME * 9) / 1250, 4);
```

Comment:
If you wrap desired looptime into function, you can even enable old behavior - just return user-configured value ...